frontend-new/styles/globals.css

```diff
@@ -5,12 +5,22 @@ body {
   padding: 0;
```

Comment:
We should move these styles into the emotion stylesheet. This will ensure that once the SSR generation is enabled, the styles will be inlined into the HTML.

Reply:
make sense, thanks